tensorflow-framework/src/main/java/org/tensorflow/framework/optimizers/AdaGrad.java

```diff
@@ -42,6 +43,9 @@
   public static final float LEARNING_RATE_DEFAULT = 0.001f;
   public static final float INITIAL_ACCUMULATOR_DEFAULT = 0.01f;
 
+  private static final ApplyAdagrad.Options[] opts = new ApplyAdagrad.Options[]{
+          ApplyAdagrad.updateSlots(true),ApplyAdagrad.useLocking(true)};
```

Comment:
Super nit: any formatter will probably complain about the missing space after a comma.